File: array/medium/majority_element_2/code_1.py
class Solution:
    # Function to find the majority elements in the array
    def findMajority(self, arr):
        #Your Code goes here.
        n = len(arr)
        req_n = n / 3
        unique_array = []
        answer_array = []
        x = 0
        
                
        for i in arr:
            if i in unique_array:
                x += 1
            else:
                current = i
                sum = 0
                for j in range(n):
                    if current == arr[j]:
                        sum += 1
                unique_array.append(current)
                if sum > req_n:
                    answer_array.append(current)
        
        answer_array.sort()
        return answer_array

File: array/medium/kadane_algorithm/code_1.c
int maxSubarraySum(int arr[], int n) {
    // Your code here
    int maxSum = arr[0];
    int currentSum = 0;
    
    for(int i = 0; i < n; i++){
        currentSum += arr[i];
        if(currentSum > maxSum){
            maxSum = currentSum;
        }
        if(currentSum < 0){
            currentSum = 0;
        }
    }
    
    return maxSum;
}